spec/rubocop/cop/rails/unknown_env_spec.rb

```diff
@@ -72,4 +83,30 @@
       Rails.env == 'production'
     RUBY
   end
+
+  context 'Rails 7.1' do
+    let(:config) do
+      RuboCop::Config.new(
+        {
+          'AllCops' => {
+            'TargetRailsVersion' => '7.1'
```

Comment:
I missed this PR and added support for `local` as part #906 as well. I think this approach is better, although it may make sense to cherry pick my commit introducing a `:rails71` shared context (or I can open a separate PR for that, but it seems small) rather than a custom config, especially given the rest of this config is just the defaults.